Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableScaleTheme.cs

```diff
@@ -59,7 +61,7 @@ public override ThemePropertyValue GetProperty(ThemeStateProperty property)
         /// <inheritdoc />
         public override void SetValue(ThemeStateProperty property, int index, float percentage)
         {
-            hostTransform.localScale = Vector3.Lerp(property.StartValue.Vector3, property.Values[index].Vector3, percentage);
+            hostTransform.localScale = Vector3.Lerp(property.StartValue.Vector3, Vector3.Scale(startScale, property.Values[index].Vector3), percentage);
```

Comment:
I wouldn't call it startScale. It's very confusing with property.StartValue.Vector3. originalScale is a bit more deliberate. 

Also RotationTheme does not take into account original rotation either. 

This change will technically "break" people who were expecting the scale value to be direct vs multiplicative....but preserving the original information is more of what people would generally want I imagine. Especially if we eventually get to the work to "reset" themes allowing all changes done to be reverted. 

Curious to hear other thoughts on this? @Julenka, @CDiaz-MS , @killerantz  ?

Reply:
naming startScale : i just copy/paste code from `ScaleOffsetColorTheme` ...
i can change both to originalScale

rotation : good point, it should be all or none

breaking : sure, it can have an impact for people that use the scale theme, on object that do not have a scale at (1,1,1). depending on the change, it will need to appear in the version updating guide.

=> my point is that currently there are some theme that behave as relative changes, and other as absolute, and this need to be changed somehow to be more uniform and consistent.
i vote for the 'relative' way ;)

Reply:
I think I agree. I think relative is better...but not sure what others would expect. More importantly though, as you said, the themes should be consistent